apps/main/jest.config.node.ts

```diff
@@ -0,0 +1,23 @@
+import type { Config } from 'jest';
+
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const preset = require('../../jest.preset.js');
+
+const nodeJestConfig: Config = {
+  ...preset,
+  displayName: 'main-node',
+  testEnvironment: 'node',
+  testMatch: ['**/next.config.node.test.ts'],
+  coverageDirectory: '../../coverage/apps/main-node',
```

Comment:
Is this the same general path as the other apps/main/jest.config.ts config coverage directory?